tests/framework/cli/test_starters.py

```diff
@@ -1395,51 +1332,27 @@ class TestToolsAndExampleFromCLI:
             "ALL",
             "TEST, LOG, DOCS",
             "test, DATA, liNt",
```

Comment:
This is the other test where we could have test cases for both accepted tool names (`test` and `tests` e.t.c.)